src/catalog/postgres/pg_core_impl.cpp

```diff
@@ -522,8 +547,10 @@ bool PgCoreImpl::CreateTableEntry(const common::ManagedPointer<transaction::Tran
   {
     col_oid_t curr_col_oid(1);
     for (auto &col : schema.GetColumns()) {
-      auto success = CreateColumn(txn, table_oid, curr_col_oid++, col);
+      col_oid_t col_oid(curr_col_oid++);
+      auto success = CreateColumn(txn, table_oid, col_oid, col);
       if (!success) return false;
+      CreateColumnStatistic(txn, table_oid, col_oid, col);
```

Comment:
Must column statistics be created alongside a column?

Reply:
We did not know where else to create it. Do you have any suggestions?